charts/urlaubsverwaltung/README.md

```diff
@@ -64,107 +61,72 @@ It is used for the following values:
-Please refer to this chart for additional MariaDB configuration options.
-
+By default, Bitnami's [PostgreSQL](https://github.com/bitnami/charts/tree/master/bitnami/postgresql) chart is deployed
+and used as database.
+Please refer to this chart for additional PostgreSQL configuration options.
 
 #### Using external Database
 
 ```yaml
 extraEnv: |
     - name: spring.datasource.url
-      value: jdbc:mariadb://external-database:3306/urlaubsverwaltung
+      value: jdbc:postgresql://external-database:5435/urlaubsverwaltung
 
 extraEnvFrom: |
     - secretRef:
         name: '{{ include "urlaubsverwaltung.fullname" . }}-database'
 
 secrets:
-  database:
-    stringData:
-      spring.datasource.username: '{{ .Values.db.username }}'
-      spring.datasource.password: '{{ .Values.db.password }}'
+    database:
+        stringData:
+            spring.datasource.username: '{{ .Values.db.username }}'
+            spring.datasource.password: '{{ .Values.db.password }}'
 ```
 
-
 ### Authentication
 
-
-#### LDAP
-
-```yaml
-extraEnv: |
-    - name: UV_SECURITY_AUTH
-      value: ldap
-    - name: UV_SECURITY_DIRECTORYSERVICE_LDAP_URL
-      value: "ldap://ldap.example.org/"
-    - name: UV_SECURITY_DIRECTORYSERVICE_LDAP_BASE
-      value: dc=example,dc=org
-    - name: MANAGEMENT_HEALTH_LDAP_ENABLED
-      value: "false"
-
-extraEnvFrom: |
-  - secretRef:
-      name: '{{ include "urlaubsverwaltung.fullname" . }}-ldap-manager'
-
-secrets:
-  ldap-manager:
-    stringData:
-      UV_SECURITY_DIRECTORYSERVICE_LDAP_MANAGER_DN: '{{ .Values.ldap.manager.dn }}'
-      UV_SECURITY_DIRECTORYSERVICE_LDAP_MANAGER_PASSWORD: '{{ .Values.ldap.manager.password }}'
-```
-
-###### LDAP Sync
+#### OpenID Connect
 
 ```yaml
 extraEnv: |
-  - name: UV_SECURITY_DIRECTORYSERVICE_LDAP_SYNC_ENABLED
-    value: "true"
-  - name: UV_SECURITY_DIRECTORYSERVICE_LDAP_SYNC_USERSEARCHBASE
-    value ou=employees,ou=accounts
-
-
-extraEnvFrom: |
-    - secretRef:
-        name: '{{ include "urlaubsverwaltung.fullname" . }}-ldap-sync'
-
-secrets:
-  ldap-sync:
-    stringData:
-      UV_SECURITY_DIRECTORYSERVICE_LDAP_SYNC_USERDN: '{{ .Values.ldap.sync.userdn }}'
-      UV_SECURITY_DIRECTORYSERVICE_LDAP_SYNC_PASSWORD: '{{ .Values.ldap.sync.password }}'
+    - name: SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_DEFAULT_AUTHORIZATION-GRANT-TYPE
+      value: "authorization_code"
+    - name: SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_DEFAULT_CLIENT-ID
+      value: $yourClientId
+    - name: SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_DEFAULT_CLIENT-SECRET
+      value: $yourClientSecret
+    - name: SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_DEFAULT_PROVIDER
+      value: "default"
+    - name: SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_DEFAULT_SCOPE
+      value: "openid,profile,email,roles"
+
+    - name: SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_DEFAULT_ISSUER-URI
+      value: $yourIssuerURI
+
+    - name: SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUER-URI
+      value: $yourIssuerURI
```

Comment:
This should be abstracted, something like `.authentication.oidc.{clientID,clientSecret,issuerURL}` and then rendered accordingly